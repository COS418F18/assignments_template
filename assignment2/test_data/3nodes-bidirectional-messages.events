send N1 N2 3
send N2 N3 2
snapshot N2
tick
send N1 N2 2
tick
send N1 N2 1
tick
send N2 N1 1
tick 3
send N3 N2 1
