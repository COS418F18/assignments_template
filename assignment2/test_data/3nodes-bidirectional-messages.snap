0
N1 4
N2 1
N3 2
N1 N2 token(3)
N1 N2 token(2)
N1 N2 token(1)
